### ISSUE TITLE
Sticky to below the top?

### ISSUE THREAD

**Issue by Musiphilos** [kuzeko/StickyHeaders#7]
Is it possible to use this code while having the sticky menu not at the top, but somewhere lower on the page, below the main menu that is fixed to the top of the page?

Thank you

**Comment by kuzeko** [comment]
Of course, in  the CSS file you need to change the value for the attribute `top` for the selector `#standing`.
